charts/incubator/frigate/questions.yaml

```diff
@@ -291,7 +291,7 @@ questions:
                         type: int
                         min: 1
                         max: 31
-                        default: 8
+                        "null": true
```

Comment:
Why are this changing?

Reply:
@stavros-k all these changes were done by you https://github.com/truecharts/charts/pull/7736/commits/79fb40ee15b6bfe514610d66b87822a2b0f11819

Some values are optional as we discovered some months ago. I had reported it, and you made the changes.